service/src/main/resources/application.yml

```diff
@@ -13,6 +13,9 @@ spring:
   # https://github.com/DataBiosphere/terra-common-lib/blob/480ab3daae282ddff0fef8dc329494a4422e32f1/src/main/java/bio/terra/common/logging/GoogleJsonLayout.java#L118
   application.name: cbas
   application.version: ${cbas.version.gitHash:unknown}
+  cache:
+    cache-names: tokenResolution
+    caffeine.spec: expireAfterWrite=1h
```

Comment:
Unfortunately it doesn't seem like there's a way to custom-set expiration based on whether the tokens have hit their expiry time, but given that Terra-UI and the to-be-implemented permission checks will both handle expired tokens it's probably fine to let the token to ID conversion hang around a little after the token itself expires.

Reply:
Yeah I think this is ok. As long as we keep the authz cache short, I don't think there's any risk here. Maybe we can rename the cache to `tokenToUserIdResolution` since there might be other token-y resolutions to cache and we don't want to re-use this one inappropriately.